source/openstack/guests.rst

```diff
@@ -35,7 +36,7 @@ Manual Guest Installation
 
 .. note:: This is only useful for manually testing some images and is no longer used for building images.
 
-Please run the following commands on one of the AARCH64 (arm64) machines directly. Please make sure you have the
+Please run the following commands on one of the machines directly. Please make sure you have the
```

Comment:
This note below is **only** for aarch64 so you need to adjust this text to mention that.